libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthPreferences.m

```diff
@@ -59,32 +58,34 @@
 // The key for storing the persisted OAuth scopes.
 NSString * const kOAuthAppName = @"oauth_app_name";
 
-
 @implementation SFSDKAuthPreferences
 
-- (void)setLoginHost:(NSString*)host {
+- (void)setLoginHost:(NSString *)host {
     NSString *oldLoginHost = [self loginHost];
-    
     if (nil == host) {
         [[NSUserDefaults msdkUserDefaults] removeObjectForKey:kSFUserAccountOAuthLoginHost];
     } else {
+
+        // Persists the login host if it doesn't exist already.
```

Comment:
@wmathurin This is the issue the test failure exposed. We weren't persisting in some cases. Also, this is the only code change from before (I don't know why `GitHub` is showing the merged changes on this PR). All tests are passing now.